frontend/src/modules/layout/components/menu/menu-quickstart.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    v-if="isPrimaryWorkspace && notcompletedGuides.length > 0"
+  >
+    <el-tooltip
+      key="menu-link-quickstart"
+      :disabled="!props.collapsed"
+      :hide-after="50"
+      content="Welcome aboard"
+      effect="dark"
+      placement="right"
+      raw-content
+    >
+      <router-link
+        id="menu-quickstart"
+        :to="{ name: 'quickstart' }"
+        class="rounded-md h-8 transition !text-gray-400 flex items-center whitespace-nowrap
+          flex-nowrap px-1.5 hover:bg-gray-50 mb-2 overflow-hidden text-sm"
+        active-class="!bg-gray-100 font-medium !text-gray-900"
+      >
+        <span class="mr-3 w-5 flex justify-center">👋</span>
```

Comment:
This should probably have a bigger width, at the moment when collapsed, it shows like this:
![Screenshot 2023-11-09 at 14 52 36](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/52c35353-36c8-46cb-8112-f7e66d5c63bb)